cmd/validate_pipeline.go

```diff
@@ -19,13 +19,14 @@ package cmd
 import (
 	"context"
 
-	"github.com/hacbs-contract/ec-cli/internal/pipeline"
-	"github.com/hacbs-contract/ec-cli/internal/policy"
 	"github.com/hashicorp/go-multierror"
 	"github.com/spf13/cobra"
+
+	"github.com/hacbs-contract/ec-cli/internal/output"
+	"github.com/hacbs-contract/ec-cli/internal/policy_source"
```

Comment:
Perhaps this would be nicer?

```suggestion
	"github.com/hacbs-contract/ec-cli/internal/policy/source"
```

Reply:
Agreed! I've updated this